docs/DEPLOY.md

```diff
@@ -257,92 +257,206 @@ There is an example nginx configuration provided at `config/deploy/nginx.conf` a
 
 > NOTE: If you've built from source, you should point the web server root directory to be `_build/prod/rel/bonfire/lib/bonfire-[current-version]/priv/static`
 
-### Nix
-
-This repo contains an experimental Flake and Nix module. These are not ready for production.
-
-Here are the detailed steps to deploy it:
-
-- run a recent version of Nix or NixOS: https://nixos.wiki
-- enable Flakes: https://nixos.wiki/wiki/Flakes#Installing_flakes
-- add `sandbox = false` in your nix.conf
-- fetch and build the app and dependencies: `nix run github:bonfire-networks/bonfire-app start_iex`
-- add it as an input to your system flake.
-- add an overlay to just the package available
-- add the required configuration in your system
-
-Your `flake.nix` file would look like the following. Remember to replace `myHostName` with your actual hostname or however your deployed system is called.
-
-```nix
-{
-  inputs.bonfire.url = "github:bonfire-networks/bonfire-app/main";
-  outputs = { self, nixpkgs, bonfire }: {
-    overlay = final: prev: with final;{
-      # a package named bonfire already exists on nixpkgs so we put it under a different name
-      elixirBonfire = bonfire.packages.x86_64-linux.default;
-    };
-    nixosConfigurations.myHostName = nixpkgs.lib.nixosSystem {
-      system = "x86_64-linux";
-      modules = [
-        {
-          environment.systemPackages = [ agenix.defaultPackage.x86_64-linux ];
-          nixpkgs.overlays = [ self.overlay ];
-        }
-        ./myHostName.nix
-        bonfire.nixosModules.bonfire
-      ];
-    };
-  };
-}
+### Guix
+
+[Guix](https://guix.gnu.org) is a functional, atomic, transactional package manager and Linux distribution. It is designed to give users more control over their general-purpose and specialized computing environments, and make these easier to reproduce over time and deploy to one or many devices.
+
+The [`bonfire-guix`](https://github.com/bonfire-networks/bonfire-guix) channel contains a Guix System service to deploy Bonfire. Assuming you have the Guix System installed on a server the following steps will allow you to have a working Bonfire instance.  You can find a more detailed guide [here](https://fishinthecalculator.me/blog/bonfire--guix-a-love-story.html).
+
+#### 1. Enable the `bonfire-guix` channel
+
+You can follow these instructions to make Guix aware of the Bonfire service. You can check you have correctly enabled `bonfire-guix` by running `guix describe`. After `guix pull``, in a new shell, your `guix describe` should look something (commit hashes will probably differ) like this:
```

Comment:
how do you enable the channel?

Reply:
Ah I forgot to link the instructions :grin: fixed, thank you